l1-contracts/contracts/bridge/L1ERC20Bridge.sol

```diff
@@ -274,9 +265,6 @@ contract L1ERC20Bridge is IL1Bridge, IL1BridgeLegacy, AllowListed, ReentrancyGua
         uint256 amount = depositAmount[_depositSender][_l1Token][_l2TxHash];
         require(amount > 0, "y1");
 
-        // Change the total deposited amount by the user
-        _verifyDepositLimit(_l1Token, _depositSender, amount, true);
```

Comment:
A point discussed with @shahar4 is that some hyperchains may need to be more "permissioned" than others and having the ability prevents some token to be bridged can most certainly be useful